rackcity/views/asset_views.py

```
@@ -164,49 +170,66 @@ def asset_add(request):  # need to make network and power connections here
     failure_message = ""
     if 'id' in data:
         failure_message += "Don't include id when adding an asset. "
-
+        return JsonResponse(
+            {"failure_message": failure_message},
+            status=HTTPStatus.BAD_REQUEST,
+        )
```

Changing this to fail early